packages/builder/src/components/common/HelpIcon.svelte

```diff
@@ -1,41 +1,163 @@
 <script>
-  import { Icon, Body } from "@budibase/bbui"
+  import {
+    FontAwesomeIcon,
+    Popover,
+    MenuItem,
+    Heading,
+    Body,
+  } from "@budibase/bbui"
+  import { auth } from "stores/portal"
+
+  $: licenseType = $auth.user?.license?.plan?.type
+  $: isPremiumUser = ["pro", "team", "business", "enterprise"].includes(
+    licenseType
+  )
+
+  let show
+  let hide
+  let popoverAnchor
 </script>
 
-<a target="_blank" href="https://github.com/Budibase/budibase/discussions">
-  <div class="inner hoverable">
-    <div class="hidden hoverable">
-      <Body size="S">Need help? Go to our forums</Body>
-    </div>
-    <Icon name="Help" size="XXL" />
-  </div>
-</a>
+<div bind:this={popoverAnchor} class="help">
+  <button class="openMenu" on:click={show}>Help</button>
+  <Popover
+    class="helpMenuPopoverOverride"
+    bind:show
+    bind:hide
+    anchor={popoverAnchor}
+  >
+    <nav>
+      <div class="header">
+        <Heading size="XS">Help resources</Heading>
+        <button on:click={hide} class="closeButton"
+          ><FontAwesomeIcon icon="fa-solid fa-xmark" /></button
```

Comment:
Odd formatting here by prettier 🤷

Reply:
Yea prettier sometimes formats like this in the first instance, but if you fix it (or structure it this way yourself in the first instance) then it won't touch it. I can't count the amount of instances of these I have fixed... They annoy me so I always ensure it's structured properly. I would encourage you to do the same in general going forward - it does help readability.